pytest-selenium/tests/conftest.py

```diff
@@ -2,6 +2,8 @@
 
 import pytest
 
+import sys
```

Comment:
These imports are in the wrong order. `import sys` should be the very first import because it is in the standard library. According to PEP 8: 

```
Imports should be grouped in the following order:
Standard library imports.
Related third party imports.
Local application/library specific imports.
You should put a blank line between each group of imports.
```
For more information please see [PEP 8](https://www.python.org/dev/peps/pep-0008/#imports) the Python Style Guide

Reply:
Updated